src/Umbraco.Web.UI/umbraco/config/lang/en.xml

```diff
@@ -170,6 +170,7 @@
     <key alias="updateDate">Last edited</key>
     <key alias="updateDateDesc" version="7.0">Date/time this document was edited</key>
     <key alias="uploadClear">Remove file(s)</key>
+    <key alias="url">URL</key>

```

Comment:
I knew about 'url' key in the 'general' section :-) but made a new one in the 'content' section just for the case of slightly different translation if needed :-)